app/models/appointment.rb

```diff
@@ -1,5 +1,23 @@
 class Appointment < ActiveRecord::Base
 	belongs_to :doctor
 	belongs_to :patient
-	validates_presence_of :doctor
```

Comment:
why removed? An appointment should definitely validate doctor, patient, schedule and status.